### PR TITLE
Alphabet

### PR DESCRIPTION
**Pull request by ArtemDomnikoff**

### CODE REVIEW

File: 4-Alphabet/Alphabet.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	counter = 0
+	result_2 = ''
+	for number in range(range_start,range_end):
+		number = chr(number)+ ' '
+		if counter % columns == 0:
+			result_2 = result_2 + '\n'
+		result_2 = result_2 + number 
+		counter = counter + 1
+	print(str(result_2))
+#print("Choose ASCII symbols and press enter: 1 - All, 2 - Latin, 3 - Cyrillic. To exit press Ctrl-C.\n")
```
Comment (daria-gh) [inline 4-Alphabet/Alphabet.py]: Does it work as expected in [issue comment](https://github.com/ArtemDomnikoff/Repository/issues/14#issuecomment-560423463)? 
@ArtemDomnikoff

**Review by daria-larionova — CHANGES_REQUESTED** [review]
Build has failed. Click on red "X" near pull request name to see the details.

File: 4-Alphabet/Alphabet.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
+def printSymbols(columns, type,alphabetRangeStart,alphabetRangeEnd):
+	counter = 0
+	result = ''
+	cyrillicMaxNumber = 64
+	latinMaxNumber = 52
+	latinNumber = 6
+	latinAlphabetRange = range(91,117)
+	for number in range(alphabetRangeStart-1, alphabetRangeEnd):
```
Comment (daria-gh) [inline 4-Alphabet/Alphabet.py]: A lot of code duplication. 
```
digit = chr(number)
				result = result + digit + ' '
				counter += 1
				if counter % columns == 0:
					result += '\n'
```

Comment (daria-gh) [reply 4-Alphabet/Alphabet.py]: This one, as example

File: 4-Alphabet/Alphabet.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+				number += latinNumber
+				digit = chr(number)
+				result = result + digit + ' '
+				counter += 1
+				if counter % columns == 0:
+					result += '\n'
+			if type == 2:
+				if counter == latinMaxNumber:
+					break
+			if type == 3:
```
Comment (daria-gh) [inline 4-Alphabet/Alphabet.py]: What are 1, 2, 3? Magical numbers. We avoid them.

**Review by daria-larionova — CHANGES_REQUESTED** [review]

**Pull request closed without merging**